### DISCUSSION TITLE
【API】我该如何在页面中调用当前页面评论数

### DISCUSSION THREAD

**Discussion by awaae001** [ArtalkJS/Artalk#947] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/a6b90726-7260-4ea4-9fee-f4a7cb75fab9)

如图所示：我该如何使用API调用评论数？

我期望可以在不同的页面调用api,而不是依靠官网的：
```html
浏览量：<span class="artalk-pv-count"></span>
评论数：<span class="artalk-comment-count"></span>
```

artalk: https://talk.m-c.top/sidebar/#/login

**Comment by qwqcode** [comment, marked as answer]
https://artalk.js.org/guide/frontend/pv.html
